core/src/test/java/org/testcontainers/junit/GenericContainerRuleTest.java

```diff
@@ -102,6 +104,15 @@ public static void setupContent() throws FileNotFoundException {
             .withCommand("/bin/sh", "-c", "while true; do cat /content.txt | nc -l -p 80; done");
 
     /**
+     * Map a file on the classpath to a file in the container, and then expose the content for testing.
+     */
+    @ClassRule
+    public static GenericContainer alpineClasspathResourceSelinx = new GenericContainer("alpine:3.2")
+            .withExposedPorts(80)
+            .withClasspathResourceMapping("mappable-resource/test-resource.txt", "/content.txt", READ_WRITE, SHARED)
+            .withCommand("/bin/sh", "-c", "while true; do cat /content.txt | nc -l -p 80; done");
```

Comment:
Would be helpful to test SELinux mode. Right now these new tests will pass even if we drop selinuxContext passing.